Fireworks/FWInterface/interface/FWFFLooper.h

```diff
@@ -80,7 +80,7 @@ class FWFFLooper : public CmsShowMainBase, public edm::EDLooperBase
    void requestChanges(const std::string &, const edm::ParameterSet &);
 
    void remakeGeometry(const DisplayGeomRecord& dgRec);
-
+   static const edm::EventSetup* m_setup;
```

Comment:
Non-const static class members are not multithread safe. Can EventSetup defined locally in the thread be delivered to the required consumers in the thread using parameters or other multithread safe means?